docs/contribution_guide/index.rst

```diff
@@ -0,0 +1,18 @@
+.. _contribution_guide:
+
+Contribution Guide
+==================
+
+This project welcomes contributions and suggestions. For information on our
+community, such as our :ref:`code_of_conduct`, :ref:`communication`, and
+regular :ref:`developer_calls`, refer to the :ref:`community` pages.
+
+.. toctree::
+   :maxdepth: 3
+
+   ways_to_contribute
+   fairlearn_repository_overview
+   fairlearn_proposals
+   contributing_code
+   contributing_example_notebooks
+   releases
```

Comment:
The order of items should better match how they are listed in "Ways to contribute". How about the following changes:
* `releases` should be the last section of `development_process`
* `fairlearn_proposals` should be the last item on this list